### PR TITLE
[WFLY-19910] Add logicalVersion to WildFly manifests

### PR DESCRIPTION
**Pull request by spyrkob**
Issue: https://issues.redhat.com/browse/WFLY-19910

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -579,7 +579,11 @@
         <version.org.ow2.asm>9.7.1</version.org.ow2.asm>
         <version.org.reactivestreams>1.0.4</version.org.reactivestreams>
         <version.org.wildfly.clustering>1.1.2.Final</version.org.wildfly.clustering>
+<<<<<<< HEAD
         <version.org.wildfly.core>27.0.0.Beta1</version.org.wildfly.core>
+=======
+        <version.org.wildfly.core>27.0.0.Beta1-SNAPSHOT</version.org.wildfly.core>
+>>>>>>> 4a3e0fab1f5 ([WFLY-19910] Add logicalVersion to WildFly manifests)
```
Comment (rhusar) [inline pom.xml:586]: Rebase went bad.

**Review by rhusar — CHANGES_REQUESTED** [review]

File: preview/channel/pom.xml [hunk 2]
```diff
@@ -160,6 +160,7 @@
                     <manifestPath>${project.build.directory}/assembly-manifest-rename/assembly/${project.artifactId}-${project.version}-manifest.yaml</manifestPath>
                     <manifestName>WildFly preview manifest version ${project.version}</manifestName>
                     <manifestId>${project.groupId}:${project.artifactId}</manifestId>
+                    <manifestLogicalVersion>WildFly preview ${project.version}</manifestLogicalVersion>
```
Comment (rhusar) [inline preview/channel/pom.xml:163]: We are calling this distribution 'WildFly Preview' - notice the capital P.

**Review by rhusar — CHANGES_REQUESTED** [review]

**Comment by bstansberry** [comment]
@spyrkob @jamezp I suspect there's a needed update to the WildFly Maven Plugin to handle the new manifest schema

**Comment by jamezp** [comment]
I just released 5.1.0.Alpah2 today. I hope that's enough.

I'd really like to find a way where we can release these types of things independently.

**Comment by spyrkob** [comment]
@bstansberry I think this should be ready now

**Comment by jfdenise** [comment]
I have logged https://issues.redhat.com/browse/WFLY-20028 to track the WildFly Galleon Plugins upgrade to 7.3.1.Final

File: pom.xml [hunk 3]
```diff
@@ -334,12 +334,12 @@
         <version.org.wildfly.bom-builder-plugin>2.0.8.Final</version.org.wildfly.bom-builder-plugin>
         <version.org.wildfly.checkstyle-config>1.0.8.Final</version.org.wildfly.checkstyle-config>
         <version.org.wildfly.common>1.6.0.Final</version.org.wildfly.common>
-        <version.org.wildfly.galleon-plugins>7.2.1.Final</version.org.wildfly.galleon-plugins>
+        <version.org.wildfly.galleon-plugins>7.3.1.Final</version.org.wildfly.galleon-plugins>
```
Comment (mchoma) [inline pom.xml:337]: I was told by @fabiobrz and @jbliznak that we are waiting for this line to be merged to ublock our testing with wildfly snapshot.

Please can we work to merge this and unblock wildfly snapshot testing.

**Review by jamezp — APPROVED** [review]

**Pull request merged**